addon/models/property.js

```diff
@@ -14,6 +14,14 @@ export default class Property {
     return this._property.type;
   }
 
+  get title() {
+    return this._property.title;
+  }
+
```

Comment:
This is used for presentation purposes only--it's used to render a label for each `schema-field`.